ocrd/ocrd/processor/base.py

```diff
@@ -65,7 +66,15 @@ def __init__(
         self.parameter = parameter
 
     def show_help(self):
-        print(generate_processor_help(self.ocrd_tool))
+        doc = ''
```

Comment:
Can you move the docstring logic to generate_processor_help, passing a `processor_module` argument to it? That way all help generating code is in one place.

Reply:
What do you mean by `processor_module`, the **instance** (`self`) of the Processor implementation? (Because that's what it would take – `generate_processor_help` only has the tool json, but we need to access all the implementation's docstring attributes.)

Reply:
Yes, `self`:

```python
generate_processor_help(ocrd_tool, processor_instance=self)
```

resp.

```python
def generate_processor_help(ocrd_tool, processor_instance=None):
  # ...
  if processor_instance:
    module = inspect.getmodule(processor_instance)
    # ...
```

Reply:
Done!